x/ibc/02-client/client/cli/tx.go

```diff
@@ -0,0 +1,155 @@
+package cli
+
+import (
+	"fmt"
+	"io/ioutil"
+	"strings"
+
+	"github.com/spf13/cobra"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/context"
+	"github.com/cosmos/cosmos-sdk/codec"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/version"
+	"github.com/cosmos/cosmos-sdk/x/auth"
+	"github.com/cosmos/cosmos-sdk/x/auth/client/utils"
+	"github.com/cosmos/cosmos-sdk/x/ibc/02-client/exported"
+	"github.com/cosmos/cosmos-sdk/x/ibc/02-client/types"
+)
+
+// GetTxCmd returns the transaction commands for IBC Clients
+func GetTxCmd(storeKey string, cdc *codec.Codec) *cobra.Command {
+	ics02ClientTxCmd := &cobra.Command{
+		Use:                        "client",
+		Short:                      "Client transaction subcommands",
+		DisableFlagParsing:         true,
+		SuggestionsMinimumDistance: 2,
+	}
+
+	ics02ClientTxCmd.AddCommand(client.PostCommands(
+		GetCmdCreateClient(cdc),
+		GetCmdUpdateClient(cdc),
+	)...)
+
+	return ics02ClientTxCmd
+}
+
+// GetCmdCreateClient defines the command to create a new IBC Client as defined
+// in https://github.com/cosmos/ics/tree/master/spec/ics-002-client-semantics#create
+func GetCmdCreateClient(cdc *codec.Codec) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "create [client-id] [path/to/consensus_state.json]",
```

Comment:
This command should take either:
- The `json` emitted by `gaiacli q ibc client consensus-state`
- A file path to the data emitted by `gaiacli q ibc client consensus-state`

Maybe something like:

```go
			var state exported.ConsensusState
			if err := cdc.UnmarshalJSON([]byte(args[1]), &state); err != nil {
				fmt.Fprintf(os.Stderr, "failed to unmarshall input into struct, checking for file...")
				contents, err := ioutil.ReadFile(args[1])
				if err != nil {
					return fmt.Errorf("error opening state file: %v\n", err)
				}
				if err := cdc.UnmarshalJSON(contents, &state); err != nil {
					return fmt.Errorf("error unmarshalling state file: %v\n", err)
				}
			}
```